### DISCUSSION TITLE
Is there a way to reasonably check (from within the runtime) the "version" of the socket.io that is running on some machine?

### DISCUSSION THREAD

**Discussion by christopherreay** [socketio/socket.io#4908] [Q&A]
Is there an explicit function for this? like what you would see in package.json?

or should I maybe implement a feature detection type of a way?

What do you think would be best?

**Comment by darrachequesne** [comment]
Hi! There is currently no explicit way to do this.

As a workaround: `require("./node_modules/socket.io/package.json").version`
